### PR TITLE
[BE] Preparing environment for tests

### PR DESCRIPTION
**Pull request by MihaiDruta**
[BE] Preparing environment for tests
https://trello.com/c/Hz9GcyOy/93-be-preparing-environment-for-tests

1.Installed LiipTestFixturesBundle and LiipFunctionalTestBundle
2.Configured new SQLite database for tests
3.First functional test on successful registering

### CODE REVIEW

**Review by nspc97 — APPROVED** [review]

File: config/packages/test/framework.yaml [hunk 1]
```diff
@@ -2,3 +2,5 @@ framework:
   test: true
   session:
     storage_id: session.storage.mock_file
+
+liip_functional_test: ~
```
Comment (SpartakusMd) [inline config/packages/test/framework.yaml]: Move this to a separate yaml file.

File: tests/RegisterTest.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        $client->request(
+            'POST',
+            '/api/register',
+            [],
+            [],
+            ['CONTENT_TYPE' => 'application/json'],
+            json_encode($data)
+        );
+
+        $this->assertEquals(200, $client->getResponse()->getStatusCode());
```
Comment (SpartakusMd) [inline tests/RegisterTest.php:41]: Check that the data in the database is present and is correct.

**Review by SpartakusMd — CHANGES_REQUESTED** [review]

File: tests/RegisterTest.php [hunk 3]
```diff
@@ -34,5 +39,32 @@ public function testSuccessfullRegister()
...
+            'name' => 'TestName',
+            'surname' => 'TestSurname'
+        ));
+
+        $this->assertCount(1, $users);
+
+        /** @var User $user */
+        $user = $repository->findOneBy(array(
+            'username' => 'TestUsername',
+            'email' => 'testemail@test.mail',
```
Comment (SpartakusMd) [inline tests/RegisterTest.php]: Search only by username as it is unique.

File: tests/RegisterTest.php [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        );
+
+        $this->assertEquals(200, $client->getResponse()->getStatusCode());
+
+        $kernel = self::bootKernel();
+        $this->entityManager = $kernel->getContainer()->get('doctrine')->getManager();
+
+        $repository = $this->entityManager->getRepository(User::class);
+
+        $users = $repository->findBy(array(
```
Comment (SpartakusMd) [inline tests/RegisterTest.php]: Why 2 queries? Use only the second one with only username as search parameter.

**Review by SpartakusMd — CHANGES_REQUESTED** [review]

**Pull request merged**